cmd/query.go

```diff
@@ -61,7 +61,7 @@ https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-
 			rc = check.Critical
 		} else if total >= cliQueryConfig.Warning {
 			rc = check.Warning
-		} else if total == 0 {
+		} else if total >= 0 {
 			rc = check.OK
 		}
```

Comment:
Seems like we can leave the whole last `else` block out.

Reply:
I've implemented the same "logic" and it should do the trick. If the value is lesser than WARNING